extendeddatacrossword.go

```diff
@@ -52,29 +49,13 @@ func (e *ErrByzantineCol) Error() string {
 // prior to the Byzantine row or column being repaired, and the Byzantine row
 // or column prior to repair is returned in the error with missing shares as
 // nil.
-func RepairExtendedDataSquare(
+func (eds *ExtendedDataSquare) RepairExtendedDataSquare(
```

Comment:
Let's name it just ‘Repair’. The struct itself already mentions that it is an extended data sqaure

Reply:
Fixed in [54b8499](https://github.com/celestiaorg/rsmt2d/pull/79/commits/54b8499ca6ab761069e30a7b79b4025035984db4)